tests/test_imports.py

```diff
@@ -4,6 +4,4 @@ class TestImport(unittest.TestCase):
     # Basic import tests for packages without any.
     def test_basic(self):
         import bq_helper
-        import cleverhans
         import tensorflow_datasets
-        import segment_anything
```

Comment:
Did we drop segment_anything?

Reply:
my bad, i may have mixed it up with another package. usage seems decent to keep atm